modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,125 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
 
+On the one hand, APIkit currently supports the following elements:
+
+* System query options:
+** $filter
+** $select
+** $search
+** $orderBy
+** $skip
+** $top
+* Single entity
+* Entity relation
+* Server-side pagination
+
+On the other hand, APIkit does not currently support the following elements:
+
+* Parameter Aliases
+* Built-in Filter Operations:
+** has
+** divby 
+* Built-in Query Functions, all but:
+** contains
+** startswith
+** endwith
+* $apply
+* $compute
+* $expand
+
+=== How does transform to SQL select work in OData 4?
+
+The OData protocol defines a series of conventions that allow clients to query identified resources using URLs. The list of queryable entities, their fields, and the relationships between them are defined in a data model using simple HTTP messages.
+As OData shares some similarities with Java database connectivity (JDBC), the `transform-to-sql-select` operation takes advantage of such similarities to help the user create SQL queries for those resources. Then, the query is returned at the payload along with the query parameters at the attributes.
+
+=== How do I implement the transform to SQL select operation using APIkit for OData 4?
+
+<apikit-odata:transform-to-sql-select> is the name of the XML element that represents this operation, which has the following parameters: 
+
+|====
+| *Parameter name* | *Description*
+| Display name | This parameter configures the operation name.
+| Module configuration | Name of the related-module configuration.
+| Entity set name | The name of the entity set that you are trying to fetch #[attributes.entitySetName].
+| Page size | A number that defines the amount of results per page during server-side-pagination.
+| Entity type name | A string obtained from the OData request using #[attributes.odataRequestAttributes], which is specified by the entity data model.
+| Entity type fields | An array of strings obtained from the OData request using #[attributes.odataRequestAttributes], which provides the list of property fields from the entity type.
+| Entity type keys | An object obtained from the OData request using #[attributes.odataRequestAttributes], which provides the map of entity key names and values.
+| System query options | Parameter obtained from the OData request using #[attributes.odataRequestAttributes].
+| Search transformation enabled | Parameter to define the behavior of the transformation.
+| Strict search term match | Parameter to define the behavior of the transformation.
+|====
+
+=== $filter expressions in OData 4 
+
+​​The `$filter` expression in OData uses comparison, logical, arithmetic, and grouping operators. You can access the list operators with their corresponding meaning at http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations[the OData4 Specification]. As mentioned before, APIkit does not support `has` and `divby` operators.
```

Comment:
-[the OData4 Specification] > [the OData4 Specification^]
Add **^** when it is an external link so it displays an icon next to the URL. 
- As mentioned before, > Remove this, start the sentence directly with "APIkit does not...

Reply:
Thanks for the ^ tip! 
Phrase removed!
;)